src/configuration/env.js

```diff
@@ -0,0 +1,15 @@
+export const get_env = async () => {
+  const {
+    REACT_APP_CONFIG_ENDPOINT: configEndpoint,
+    REACT_APP_CONFIG_PATH: mockConfigPath,
+    REACT_APP_STATE_ENDPOINT: initStateEndpoint,
+    REACT_APP_MOCK_STATE_PATH: mockInitState,
+    REACT_APP_JOAL_CONF_ENDPOINT: joalConfigEndpoint,
+    REACT_APP_MOCK_JOAL_CONF_PATH: mockJoalConfigPath,
+  } = process.env;
```

Comment:
Can we have some other source than `process.env` ? The app will have to run on *non-server* environment where setting environment variable is not an option.

Reply:
Yes we can but that mean to hard code the url of the configuration inside the code